Dockerfile

```diff
@@ -21,6 +21,18 @@ ENV PATH=${PATH}:/go/src/github.com/docker/app/bin/
 ARG DEP_VERSION=v0.5.0
 RUN curl -o /usr/bin/dep -L https://github.com/golang/dep/releases/download/${DEP_VERSION}/dep-linux-amd64 && \
     chmod +x /usr/bin/dep
+ARG GOTESTSUM_VERSION=v0.3.4
+RUN mkdir $GOPATH/src/gotest.tools && \
```

Comment:
Can't we use `go get` here directly?

Reply:
It doesn't support the `-o` option which we use below (neither does `go install`)

I also tried just fetching the source using `go get` and then build manually but couldn't figure out how to do that in `GO111MODULE=on` mode and without that mode the `@version` syntax isn't allowed.

Reply:
Right 👍